### DISCUSSION TITLE
Login using Postman

### DISCUSSION THREAD

**Discussion by phantomate** [TandoorRecipes/recipes#1691] [Q&A] [answered]
Hi folks,

do you know, how do I login on my local instance using postman?
Im using POST for http method, /accounts/login for url and login (username) + password as params. Am I missing something?
I tried to use my browser dev tools to inspect the request but nothing is shown for /accounts/login.
I hope you can help me.

**Comment by vabene1111** [comment, marked as answer]
you should use token authentication. On your user account settings there is an API tab where the token and header is shown.

**Reply by phantomate** [reply]
thank you it works like a charm
